data/huion-h640p.tablet

```diff
@@ -0,0 +1,36 @@
+# HUION
+# H640P
+# 
+#      *-----------------------------------*
+#    A |                                   |
+#    B |                                   |
+#    C |                                   |
+#      |              DISPLAY              |
+#    D |                                   |
+#    E |                                   |
+#    F |                                   |
+#      *-----------------------------------*
+#
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/145
+
+[Device]
+Name=Huion H640P
+ModelName=
+DeviceMatch=usb:256c:006d
```

Comment:
as you noticed, `006d` is already in use (because HUION re-uses USB ids). You need to add the name to the match string as well. See `data/huion-h610-pro.tablet` for a device that already uses this approach.